sechub-sereco/src/test/java/com/daimler/sechub/sereco/importer/SarifV1JSONImporterTest.java

```diff
@@ -0,0 +1,155 @@
+import com.daimler.sechub.sereco.metadata.SerecoMetaData;
+import com.daimler.sechub.sereco.metadata.SerecoSeverity;
+import com.daimler.sechub.sereco.metadata.SerecoVulnerability;
+import com.daimler.sechub.sereco.test.SerecoTestFileSupport;
+
+public class SarifV1JSONImporterTest {
+
+    private SarifV1JSONImporter importerToTest;
+    private String sarifBrakeman;
+    private String sarifEmptyResult;
+    private String sarifThreadflowsExample;
+    
+    @Before
+    public void before() {
+        importerToTest = new SarifV1JSONImporter();
+        sarifBrakeman = SerecoTestFileSupport.INSTANCE.loadTestFile("sarif/sarif_2.1.0_brakeman.json");
+        sarifEmptyResult = SerecoTestFileSupport.INSTANCE.loadTestFile("sarif/sarif_2.1.0_empty_results.json");
+        sarifThreadflowsExample = SerecoTestFileSupport.INSTANCE.loadTestFile("sarif/sarif_2.1.0_threadflows_example.json");
+    }
+
+    @Test
+    public void sarifreport_can_be_imported() {
+        /* prepare */
+
+        ImportParameter paramBrakeman = ImportParameter.builder().importData(sarifBrakeman).importId("id1").productId("SARIF").build();
+        ImportParameter paramThreadFlows = ImportParameter.builder().importData(sarifThreadflowsExample).importId("id1").productId("SARIF").build();
+
+        /* execute */
+        ProductImportAbility ableToImportBrakemanSarif = importerToTest.isAbleToImportForProduct(paramBrakeman);
+        ProductImportAbility ableToImportThreadFlowSarif = importerToTest.isAbleToImportForProduct(paramThreadFlows);
+
+        /* test */
+        assertEquals("Was able to import sarif!", ProductImportAbility.ABLE_TO_IMPORT, ableToImportBrakemanSarif);
+        assertEquals("Was able to import sarif!", ProductImportAbility.ABLE_TO_IMPORT, ableToImportThreadFlowSarif);
+    }
+
+    @Test
+    public void empty_sarifreport_throws_exception() {
+        
+        /* prepare */
+        /* test */
+        assertThrows(IOException.class, () -> {
+            importerToTest.importResult(null);
+        });
+        
+        assertThrows(IOException.class, () -> {
+            importerToTest.importResult("");
+        });
+    }
+    
+    @Test
+    public void sarifreport_has_errorlevel() throws Exception {
+        /* prepare */
+        SerecoMetaData result = importerToTest.importResult(sarifBrakeman);
+
+        /* execute */
+        List<SerecoVulnerability> vulnerabilities = result.getVulnerabilities();
+        SerecoVulnerability v1 = vulnerabilities.get(0);
+
+        /* test */
+        assertEquals(SerecoSeverity.HIGH, v1.getSeverity());
+    }
+
+    @Test
+    public void sarifreport_has_no_description() throws Exception {
+        /* prepare */
+        SerecoMetaData result = importerToTest.importResult(sarifBrakeman);
+
+        /* execute */
+        List<SerecoVulnerability> vulnerabilities = result.getVulnerabilities();
+        SerecoVulnerability v1 = vulnerabilities.get(0);
```

Comment:
Please use vulnerability1 instead of just v1.

Reply:
done